### ISSUE TITLE
[German] Ampersands in German locales

### ISSUE THREAD

**Issue by tillheilmann** [citation-style-language/locales#163]
The German locales use ampersands for joining multiple roles, e.g. "Herausgeber & Übersetzer" for editortranslator. Actually, ampersands should be used in the context of business only. So, traditionally, one would expect "Herausgeber und Übersetzer" or "Hrsg. u. Übers." for the short form (i.e. using the very German "and" term as it is defined in the locales themselves).

**Comment by bwiernik** [comment]
@adam3smith Should any changes be made to the default German locale on this issue?

**Comment by tillheilmann** [comment]
[German Wikipedia on "Et-Zeichen"](https://de.wikipedia.org/wiki/Et-Zeichen) (ampersand) references Duden (2017) in explaining that ampersands should only be used in names of businesses and derived titles, headings etc.:

**Comment by adam3smith** [comment]
Yeah, sorry I never responded to that -- removing the ampersand for anything but and="symbol" (which I think isn't currently configurable anyway?) as the default in German (I believe for all three locales) makes sense.

**Comment by POBrien333** [comment]
Is this issue still valid or can it be closed?

**Comment by tillheilmann** [comment]
It seems the ampersands are still in the locales, so I guess it's still valid. If you want me to make the changes to the 3 locales, I will gladly do so. But I am an absolute idiot when it comes to GitHub, so I appreciate advice on how to proceed.
